stable/rabbitmq/templates/statefulset.yaml

```diff
@@ -134,6 +134,10 @@ spec:
         volumeMounts:
           - name: config-volume
             mountPath: /opt/bitnami/rabbitmq/conf
+          {{- if .Values.rabbitmq.ssl.enabled }}
+          - name: config-certs
+            mountPath: /opt/bitnami/rabbitmq/certs
+          {{ end }}
```

Comment:
We should also support the possibility of re-using already existing secrets with the certificates. Check for instance the [MongoDB chart](https://github.com/helm/charts/tree/master/stable/mongodb) and the parameter `existingSecret`

Reply:
I added the possibility to use a pre existing secrets